hack/arktos-up.sh

```diff
@@ -18,6 +18,10 @@ PARTITION_CONFIG_DIR=${PARTITION_CONFIG_DIR:-"/var/run/kubernetes/"}
 
 VIRTLET_METADATA_DIR=${VIRTLET_METADATA_DIR:-"/var/lib/virtlet"}
 VIRTLET_LOG_DIR=${VIRTLET_LOG_DIR:-"/var/log/virtlet"}
+VIRTLET_DEPLOYMENT_FILES_DIR=${VIRTLET_DEPLOYMENT_FILES_DIR:-"/tmp"}
+
+# Default to vm-runtime master branch for now till we have a release and versioning setup
+VIRTLET_DEPLOYMENT_FILES_SRC=${VIRTLET_DEPLOYMENT_FILES_SRC:-"https://raw.githubusercontent.com/futurewei-cloud/arktos-vm-runtime/master/deploy"}
```

Comment:
You probably want to add a version here. Otherwise in future any change in arktos-vm-runtime repo will break early versions of Arktos.

Reply:
Agreed we need to get versioning support finalized to get a refined support ( as the comment mention ). current thinking for this PR is that:
1. branch implicitly carries major release versions. i.e. when we cut (V1 release, or for example, the 330 release ), both Arktos and Arktos runtime will be forked to 330 release branch and the runtime deployment source files will be persevered there.
2. for minor release or critical bug fixes. in dev branch,  they can be sync-ed with each dev branch and master branches should be always in sync.  cherry-picks can be made into released branches as bug fix as well on a needed basis.
3. once we have a release site with finalized release story, those files will be tar-ed into tarball with major and minor versions, along with the binaries. 

#3 can be done post 330 release. current PR should support #1 and #2.

Reply:
For 1) and 2), I don't think these two repos should be coupled so tight. According to the description things will break if two repos have different branch policies.  These two repos could have two different group of developers. They should only reply on some simple public locations, such as  deployment/v1/xxx.yaml

Reply:
as we discussed, a release branch release-330 was created at the vm-runtime repo. will change the default in Arktos to use this current release.